interface/themes/style_manila.scss

```diff
@@ -464,13 +464,57 @@ body dl {
 
 /* main.php*/
 .body_main{
-  background-color:#C9DBF2;
+  background-color:#F4F4F4;
 }
 /*~~openemr5 tabs menu , user and logout icon~~*/
 #menu_icon, #logout_icon, #user_icon {
-    color:#676666 !important;
+    color:#9c9c9c !important;
```

Comment:
You use #9c9c9c in a lot of places in this file.  Should probably be pulled into a variable.